rust/lance-index/src/vector/transform.rs

```diff
@@ -167,6 +167,26 @@ impl Transformer for KeepFiniteVectors {
     }
 }
 
+#[derive(Debug)]
+pub struct RemoveColumn {
+    column: String,
+}
+
+impl RemoveColumn {
+    pub fn new(column: &str) -> Self {
+        Self {
+            column: column.to_owned(),
+        }
+    }
+}
+
+#[async_trait]
+impl Transformer for RemoveColumn {
+    async fn transform(&self, batch: &RecordBatch) -> Result<RecordBatch> {
+        Ok(batch.drop_column(&self.column)?)
```

Comment:
What is the behavior of `column not exist` ? Let caller handle it?

Reply:
it just clones the `RecordBatch` and no error returned